### DISCUSSION TITLE
How to change the color of the loading bubble?

### DISCUSSION THREAD

**Discussion by palm-ER** [FlowiseAI/Flowise#2066] [Q&A]
<img width="113" alt="bubble" src="https://github.com/FlowiseAI/Flowise/assets/104501732/8b4d273d-b4cf-4db7-acc9-bd67e8cc71aa">

This thing.

**Comment by HenryHengZJ** [comment]
you can modify the source code here on your forked repo - https://github.com/FlowiseAI/FlowiseChatEmbed/blob/main/src/components/TypingBubble.tsx
